lua/efmls-configs/linters/ruff.lua

```diff
@@ -0,0 +1,22 @@
+-- Metadata
+-- languages: python
+-- url: https://github.com/astral-sh/ruff
+
+local sourceText = require('efmls-configs.utils').sourceText
+local fs = require('efmls-configs.fs')
+
+local linter = 'ruff'
+local command = string.format('%s check --output-format', fs.executable(linter))
```

Comment:
The `--output-format` here seems to be missing a value.

Also I see that we can use stdin if we provide `--stdin-filename`, so you can add that at the end of the command:

```
%s check --output-format text --stdin-filename "${INPUT}"
```

Reply:
Yes, I wanted to use the `--output-format=pylint` so that I could just copy-paste the `lintFormats` from pylint.
However, the difference is minor so I may just adapt the `lintFormats`.


```
ipykernel (217da9f)  3.11.5  took 24s 
❯ ruff check .                                  
ipykernel/tests/test_io.py:211:13: PLW1510 `subprocess.run` without explicit `check` argument
ipykernel/tests/test_kernel.py:255:12: PLW1510 `subprocess.run` without explicit `check` argument
Found 2 errors.

ipykernel (217da9f)  3.11.5  
✗ ruff check --output-format=pylint . 
ipykernel/tests/test_io.py:211: [PLW1510] `subprocess.run` without explicit `check` argument
ipykernel/tests/test_kernel.py:255: [PLW1510] `subprocess.run` without explicit `check` argument
```

Reply:
pylint doesn't seem to provide the column position 🤔 but up to you if you want to parse pylint messages or just plain messages. The goal is to show those messages on the neovim side either way.

Reply:
Let's go for the original one (without `--output-format=pylint`).
However, I would need some help to craft the `lintFormats` accordingly.

Reply:
The following should work: `%.%#:%l: [%t%n] %m`

Reply:
Ok, I added it :)

Reply:
Just need you to add `--outut-format=pylint` and we should be good to go 😅

Reply:
Well, no, I thought we were sticking with the regular output. Oh, then I should change the `lintFormats` back.
Sorry for the misunderstanding.

Reply:
I want to go with the first one of those two (if it is OK for you):
```
ipykernel (217da9f)  3.11.5  took 24s 
❯ ruff check .                                  
ipykernel/tests/test_io.py:211:13: PLW1510 `subprocess.run` without explicit `check` argument
ipykernel/tests/test_kernel.py:255:12: PLW1510 `subprocess.run` without explicit `check` argument
Found 2 errors.

ipykernel (217da9f)  3.11.5  
✗ ruff check --output-format=pylint . 
ipykernel/tests/test_io.py:211: [PLW1510] `subprocess.run` without explicit `check` argument
ipykernel/tests/test_kernel.py:255: [PLW1510] `subprocess.run` without explicit `check` argument
```

Reply:
Oh whoops actually that's my bad, I thought we were going for pylint style messages lol

In that case, disregard my previous message and use this for the lintFormats: `%.%#:%l:%c: %t%n %m`

Reply:
Done !